src/events/deposits.upsert.service.ts

```diff
@@ -255,4 +257,37 @@ export class DepositsUpsertService {
       );
     });
   }
+
+  async syncDepositedIron(): Promise<void> {
+    const aggregate = await this.prisma.deposit.aggregate({
+      _sum: {
+        amount: true,
+      },
+    });
+    this.influxDbService.writePoints([
+      {
```

Comment:
For influxdb, will this `measurement` be autocreated as a category when the first measurement is sent? Or do I define somewhere?

Reply:
You don't need to explicitly define this anywhere.